ctapipe/io/hdftableio.py

```diff
@@ -91,6 +99,16 @@ def write(self, table_name, containers):
         """
         pass
 
+    @abstractmethod
+    def open(self, filename):
```

Comment:
need to add  **kwargs here, since the parameters of the overridden methods must match the base class

Reply:
that's an old mistake, but might as well finish it now

Reply:
I this is not true. Python (at least not abc) does not check the method's signature:

```
from abc import ABCMeta, abstractmethod


class A(metaclass=ABCMeta):

    @abstractmethod
    def test(self):
        pass


class B(A):

    def test(self, name):
        print('Hello', name + '!')


b = B()
b.test('World')
```